### ISSUE TITLE
No model logs from PyTorch 1.10 SageMaker endpoint

### ISSUE THREAD

**Issue by setu4993** [aws/sagemaker-pytorch-inference-toolkit#116]
**Describe the bug**

No model logs show up in endpoints created for PyTorch 1.10. Works fine by going to PyTorch 1.9 / 1.9.1, but not with 1.10.

Dependencies install correctly, and the model loads up, but there's no logs from the container that get forwarded to CloudWatch.

I tried updating to 2.8.0 within the container but that doesn't work because the properties file is different and it fails trying to find `log4j.properties`.

**Expected behavior**

Logs should be forwarded to CloudWatch.

**Screenshots or logs**

<img width="1727" alt="Screen Shot 2022-02-25 at 13 37 19" src="https://user-images.githubusercontent.com/1833708/155806441-57a2fbce-de2a-408b-a9a6-63c16c7d6c11.png">

**System information**
A description of your system. Please provide:
- **Toolkit version**: 2.7.0
- **Framework version**: 1.10
- **Python version**: 3.8
- **CPU or GPU**: GPU
- **Custom Docker image (Y/N)**: N

**Comment by SamShowalter** [comment]
Any update on this? I am also having these issues with both torch versions 1.9 and 1.10 and my code is not compatible with the older images.

**Comment by setu4993** [comment]
@SamShowalter : I didn't get any traction here, but after raising it AWS Support, they did tell me a couple weeks ago (and I have since verified) that logs work fine for PyTorch 1.10.2 and PyTorch 1.11 images. They were released / updated in the last couple months according to this [doc](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/dlc-release-notes.html).

We also saw the same issue of missing logs on 1.9 images, but have transitioned to 1.10 / 1.11, so haven't followed up.
